docs/ops/config.md

```diff
@@ -183,6 +183,12 @@ unless user define a `OptionsFactory` and set via `RocksDBStateBackend.setOption
 
 {% include generated/rocks_db_configurable_configuration.html %}
 
+### blink table planner
```

Comment:
- this page is meant as a full reference; I'd assume these should also be placed on the API specific pages
- please follow the naming convention of other sectons, i.e. this should be `Blink Table Planner`
- I'm not sold on the placement, this area here is all about state (checkpointing, rocksdb, QS)

Reply:
- Documents about blink will be introduced later, then config will also be placed together with the related API.
- Fixed it.
- I moved these to the end of this page.